cloud/google/cmd/gce-machine-controller/main.go

```diff
@@ -55,7 +56,16 @@ func main() {
 		glog.Fatalf("Could not create client for talking to the apiserver: %v", err)
 	}
 
-	actuator, err := google.NewMachineActuator(*kubeadmToken, client.ClusterV1alpha1().Machines(corev1.NamespaceDefault), *machineSetupConfigsPath)
+	configWatch, err := machinesetup.NewConfigWatch(*machineSetupConfigsPath)
+	if err != nil {
+		glog.Fatalf("Could not create config watch: %v", err)
+	}
+	machineSetupConfigs, err := configWatch.ValidConfigs()
```

Comment:
This cannot be done here. It needs to be done on every time a machine is created. The file's contents are allowed to change during runtime and needs to be re-parsed.

Reply:
Got it, I was not understanding the "on-cluster" context in which this code runs (inside of gce-machine-controller). I will update with a version that encapsulates the refreshing logic.